### ISSUE TITLE
Text To Speech, Speech To Text

### ISSUE THREAD

**Issue by thomas992** [fubark/cosmic#45]
https://github.com/coqui-ai/STT
https://github.com/coqui-ai/TTS

Chromium does not include text to speech or speech to text. Firefox does not have it either, text to speech, speech to text are a vital part of computer interfacing.

**Comment by fubark** [comment]
You're right we need this. Another model to look into is wav2vec. Ideally we'd want it to be small enough to be embedded into apps so it would work offline and still have decent wer. I don't know how much work that would be atm.
